package.json

```diff
@@ -47,8 +47,8 @@
     "@formatjs/intl-numberformat": "^6.2.5",
     "@formatjs/intl-pluralrules": "^4.0.13",
     "@gorhom/portal": "^1.0.14",
-    "@oguzhnatly/react-native-image-manipulator": "github:Expensify/react-native-image-manipulator#5cdae3d4455b03a04c57f50be3863e2fe6c92c52",
-    "@onfido/react-native-sdk": "7.0.1",
+    "@oguzhnatly/react-native-image-manipulator": "github:Expensify/react-native-image-manipulator#c5f654fc9d0ad7cc5b89d50b34ecf8b0e3f4d050",
```

Comment:
@nkuoch is on leave till June. I'm not sure if there was a specific reason, it might've been something we accidentally overlooked 🤔 Was the solution to the regression to update the hash?